charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Datadog changelog
 
+## 2.31.0
+
+***Warning:*** From this version onwards, on GKE Autopilot, only one "datadog" Helm chart release is allowed by the Kubernetes namespace due to the following new constraints:
+* On GKE Autopilot, make the "Agent" DaemonSet serviceAccountName immutable.
+* On GKE Autopilot, make the "Install Info" ConfigMap name immutable.
```

Comment:
“immutable” could mean that the service account name and the config map name cannot be changed once the Helm chart has been installed but that it could still be chosen with different values for several Helm releases.

```suggestion
* On GKE Autopilot, hardcode the "Agent" DaemonSet serviceAccountName.
* On GKE Autopilot, hardcode the "Install Info" ConfigMap name.
```